rcsb/workflow/targets/ProteinTargetSequenceWorkflow.py

```diff
@@ -181,6 +182,10 @@ def __exportTargetsFasta(self, resourceName, useCache=True, addTaxonomy=False, r
                     if backupPharos:
                         okB = ptP.backup(self.__cfgOb, self.__configName, remotePrefix=remotePrefix, useStash=True, useGit=True)
                         logger.info("%r targets backup status (%r)", resourceName, okB)
+                elif not (reloadPharos or fromDbPharos):
```

Comment:
So basically the logic is:
- If `ptP.testCache()` succeeds, backup the Pharos data
- elif `ptP.testCache()` fails, restore from stash

Am I following that right?

Reply:
Yeah, if reloadPharos or fromDbPharos are both false, nothing really gets done inside `PharosTargetProvider`. Although, now that I'm thinking of it, perhaps just an `else` would be better. Do you think we should restore the backup if something went wrong in loading the data from the SQL dump?

Reply:
Do you mean if something goes wrong during the loading of data from the SQL dump obtained from Pharos' site directly? If so, I think in that case it's best for it to totally fail, because we would only be doing that if we want to rebuild the cache files, right?

Reply:
Good point